_includes/manuals/1.9/4.4.8.2_prov_sles.md

```diff
@@ -0,0 +1,4 @@
+The installation media URL has to contain the contents of the first SLES DVD, it's easiest to loopback mount the ISO image on a webserver. For Puppet, the `systemsmangement:puppet` repository on OBS are used.
+
+##### SLES 11
+For Puppet, in addition to `systemsmangement:puppet` also the `devel:languages:ruby:backports` repository on OBS and the SLES SDK DVD is used. The placeholder in the AutoYaST SLES template has to be updated with the actual SDK URL.
```

Comment:
Typo, `systemsmanagement:puppet` , missing an `a` in between `n` and `g`. Same on line 1